config/routes.rb

```diff
@@ -16,4 +16,5 @@
     resources :likes, only: [:create]
   end
 end
+post '/users/:user_id/posts/:post_id/comments', to: 'comments#create', as: 'user_post_comments'
```

Comment:
- [ ] Great job with your implementation of the comments. However, per the project requirements, there should be a pair of routes in the `config/routes.rb` file for creating comments similar to what you did for `Posts create` and `Posts New`. Currently, you only have the `Post` route for comments in your project :+1: 
- Example
   ```ruby
      POST   /posts/:post_id/comments(.:format)     comments#create
    GET   /posts/:post_id/comments/new(.:format)     comments#new
   ```
- [ ] After doing this, you should ensure that there is a separate form in the `views/comments/new.html.erb` file that accepts input for creating comments. :+1: